File: data-model/src/main/java/com/medziku/plantwatering/station/parts/SoilMoistureSensor.java
package com.medziku.plantwatering.station.parts;


import com.medziku.plantwatering.station.AbstractPlantStationPart;
import com.medziku.plantwatering.station.PasivePlantStationPart;

public abstract class SoilMoistureSensor extends AbstractPlantStationPart implements PasivePlantStationPart {

    protected float mSoilMoistureLevel;

    public float getSoilMoistureLevel() {
        return mSoilMoistureLevel;
    }

    public void setSoilMoistureLevel(float soilMoistureLevel) {
        this.mSoilMoistureLevel = soilMoistureLevel;
    }
}

File: data-model/src/main/kotlin/com/medziku/plantwatering/station/parts/SoilMoistureSensor.kt
package com.medziku.plantwatering.station.parts


import com.medziku.plantwatering.station.PasivePlantStationPart

abstract class SoilMoistureSensor(override var name: String, var soilMoistureLevel: Float = 0f) : PasivePlantStationPart
